tap_typeform/streams.py

```diff
@@ -292,76 +318,52 @@ def sync_forms(atx):
             continue
 
         # start_date is defaulted in the config file 2018-01-01
-        # if there's no default date and it gets set to now, then start_date will have to be
-        #   set to the prior business day/hour before we can use it.
-
+        # if there's no default date and it gets set to now
         now = datetime.datetime.now(pytz.utc)
-        if incremental_range == "daily":
-            s_d = now.replace(hour=0, minute=0, second=0, microsecond=0)
-            start_date = pendulum.parse(atx.config.get('start_date', s_d + datetime.timedelta(days=-1, hours=0)))
-        elif incremental_range == "hourly":
-            s_d = now.replace(minute=0, second=0, microsecond=0)
-            start_date = pendulum.parse(atx.config.get('start_date', s_d + datetime.timedelta(days=0, hours=-1)))
-        LOGGER.info('start_date: {} '.format(start_date))
-
-
-        # end date is not usually specified in the config file by default so end_date is now.
-        # if end date is now, we will have to truncate them
-        # to the nearest day/hour before we can use it.
-        if incremental_range == "daily":
-            e_d = now.replace(hour=0, minute=0, second=0, microsecond=0).strftime("%Y-%m-%d %H:%M:%S")
-            end_date = pendulum.parse(atx.config.get('end_date', e_d))
-        elif incremental_range == "hourly":
-            e_d = now.replace(minute=0, second=0, microsecond=0).strftime("%Y-%m-%d %H:%M:%S")
-            end_date = pendulum.parse(atx.config.get('end_date', e_d))
-        LOGGER.info('end_date: {} '.format(end_date))
+        s_d = now.replace(hour=0, minute=0, second=0, microsecond=0)
+        start_date = pendulum.parse(atx.config.get('start_date', s_d))
```

Comment:
The fallback value here is wrong

```python
>>> import pendulum
>>> pendulum.parse("2021-09-22T12:34:56Z")
DateTime(2021, 9, 22, 12, 34, 56, tzinfo=Timezone('UTC'))
>>> from datetime import datetime
>>> pendulum.parse(datetime.now())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/share/virtualenvs/tap-typeform/lib/python3.8/site-packages/pendulum/parser.py", line 29, in parse
    return _parse(text, **options)
  File "/usr/local/share/virtualenvs/tap-typeform/lib/python3.8/site-packages/pendulum/parser.py", line 45, in _parse
    parsed = base_parse(text, **options)
  File "/usr/local/share/virtualenvs/tap-typeform/lib/python3.8/site-packages/pendulum/parsing/__init__.py", line 74, in parse
    return _normalize(_parse(text, **_options), **_options)
  File "/usr/local/share/virtualenvs/tap-typeform/lib/python3.8/site-packages/pendulum/parsing/__init__.py", line 115, in _parse
    return _parse_iso8601_interval(text)
  File "/usr/local/share/virtualenvs/tap-typeform/lib/python3.8/site-packages/pendulum/parsing/__init__.py", line 215, in _parse_iso8601_interval
    if "/" not in text:
TypeError: argument of type 'datetime.datetime' is not iterable
```

Reply:
yep, adding a call to `isoformat()` to pass the default as a string